src/validation/JoiValidator.js

```diff
@@ -6,7 +6,7 @@ import Joi from '@hapi/joi';
 const JoiValidation = {
 
   validateString() {
-    return Joi.string();
+    return Joi.string().regex(/^[a-zA-Z]+$/);
```

Comment:
I think you should create another schema that will only accept *alphabet* instead of modifying this one. What do you think?

Reply:
yes that's cool... since there might be other properties that don't require to be only alphabet in future

Reply:
Exactly